test/hotspot/jtreg/compiler/loopopts/superword/TestMulAddS2I.java

```diff
@@ -36,14 +36,20 @@
 import jdk.test.lib.Platform;
 
 public class TestMulAddS2I {
```

Comment:
Might be worth to add 8325252 as `@bug` number as well since you added quite a few tests.